gem/exe/rb-sys-dock

```diff
@@ -310,8 +314,8 @@ def default_command_to_run(input_args)
   input_cmd = input_args.empty? ? "true" : input_args.join(" ")
 
   if OPTIONS[:build]
-    with_bundle = +"test -f Gemfile && bundle install && #{input_cmd} && bundle exec rake native:$RUBY_TARGET gem"
-    without_bundle = "#{input_cmd} && rake native:$RUBY_TARGET gem"
+    with_bundle = +"test -f Gemfile && bundle install && #{input_cmd} && bundle exec rake native:#{ruby_target} gem RUBY_TARGET=#{ruby_target}"
```

Comment:
I'm not quite sure why adding the environment variable at the end works, but why adding it via `export RUBY_TARGET=#{ruby_target} &&` before this line does not.

Reply:
`NAME=VALUE` feature is a well-documented `rake` feature: 
1. https://github.com/ruby/rake/blob/09a1cd41d13d0c2c23a0775f860a8c5e71d43911/doc/command_line_usage.rdoc#label-Rake+Command+Line+Usage
2. https://github.com/ruby/rake/blob/master/doc/rakefile.rdoc#label-Tasks+with+Arguments

In another Rake task, I confirmed `NAME=VALUE bundle exec rake` doesn't work.